app/views/people/_add_contact.haml

```diff
@@ -3,6 +3,6 @@
   .media
     .pull-right
       = aspect_membership_dropdown(contact, inviter, false)
-    = person_image_link(inviter, :size => :thumb_small, :class => 'img')
+    = person_image_link(inviter, :size => :thumb_small, :class => 'media-left')
```

Comment:
Given the examples on http://getbootstrap.com/components/#media I'd say this should have class `media-object`, wrapped by a `.media-left` div. Also, while touching this line, please use double quotes and the new ruby hash syntax (`size: :thumb_small, class: "media-object"`).